lib/httpServer.js

```diff
@@ -48,9 +51,15 @@ module.exports = function (provider, logger) {
             for (var i = 0; i < payload.length; i++) {
               var item = payload[ i ];
               logger.log(item.method);
+              if (item.method === 'eth_subscribe') {
+                return send400();
```

Comment:
Because of my other comment, it would be best if this returned a proper JSON-RPC error, perhaps with a message like "Method not allowed over HTTP transport."